packages/zowe-explorer/src/dataset/ZoweDatasetNode.ts

```diff
@@ -40,6 +40,7 @@ import { promiseStatus, PromiseStatuses } from "promise-status-async";
 import { getDocumentFilePath, initializeFileOpening, updateOpenFiles } from "../shared/utils";
 import { IZoweDatasetTreeOpts } from "../shared/IZoweTreeOpts";
 import { LocalFileManagement } from "../utils/LocalFileManagement";
+import { imperative } from "@zowe/cli";
```

Comment:
if possible, please import the instance of imperative provided by the `@zowe/zowe-explorer-api` package.